src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
```

Comment:
Popping the last element from `fromFormat` and `toFormat` to get the separators might not be reliable if the formats don't end with the separator. Consider explicitly passing the separator as a separate argument or ensuring the format arrays are structured to include separators as the last element.